api/cronhook/start-cron.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+# start-cron.sh
+
+printenv >> /etc/environment
```

Comment:
Django needs to talk to `dbapi` to perform correctly, and it needs credentials to do so. Those credentials are stored in the environment; however cron does not by default provide the env it is running in to the started cron jobs. Hence, we need to pass on the env to the command line django by ourselves.

I will also argue that it does not pose an additional security thread: an attacker that can read from `/etc/environment` can likely also read env variables.